app/controllers/wpcc/your_details_controller.rb

```diff
@@ -8,11 +10,13 @@ def new
 
     def create
       @your_details_form = Wpcc::YourDetailsForm.new(your_details_form_params)
+      @your_details_form = present(@your_details_form)
```

Comment:
@Tr4pSt3R sounds good, perhaps a line break after the first opening parenthesis?